### DISCUSSION TITLE
swap textures

### DISCUSSION THREAD

**Discussion by ChiroxTreichel** [pixiv/three-vrm#1380] [Help]
Hi there,

i am trying to swap texture of my Wifes VRM while it runs in three.js ...
This works ... less well than hoped.

I am able to change the Texture of the Normalmaps but i cant find the path of the other textures.

I know, their name is like "_16" ... but even this info does not help at all.

	(gltf) => {
		const model = gltf.scene;
		const vrm = gltf.userData.vrm;

		currentVrm = vrm;
		scene.add(vrm.scene);

		vrm.scene.traverse((child) => {
		  if (child.isMesh && child.material) {
			
			if (child.name === 'Hair') {
				console.log(child);
			  textureLoader.load("/new/models/<Modelname>/Textures/rot_16.png", (texture) => {
				const material = child.material;
				material.map = texture;
				material.needsUpdate = true;
			  });
			}
		  }
		});
	},

Does anyone know, where the real textures can be found?

**Comment by 0b5vr** [comment]
I think the code should work, what exactly is your problem?
It would be nice if you could provide a demo to show what is wrong with the behavior.

> i cant find the path of the other textures.

What is the "path" here? Is there a file on your server?

**Reply by ChiroxTreichel** [reply]
you can find the current state on https://models.talutah.de/new/three/index.html

The File in the path is the Hair Texture in a different color.
But the material of the Mesh "Hair" is not addressing the texture of the Hair...instead its changing the uv-map of the Hair.

I am searching for the texture to replace. Thats what i meant with path.
The original texture name created by vroid studio is counting up from _1 to _16 (in this Case)
